src/components/10-atoms/radio/index.scss

```diff
@@ -0,0 +1,190 @@
+axa-radio {
+  @include typo(primary, default);
+  display: block;
+  flex: none;
+
+  *, *::after, *::before {
+    box-sizing: border-box;
+  }
+
+  &:hover:not([disabled]):not(.a-radio--error) {
+    color: $color-axa-blue;
+
+    & .a-radio__icon {
+      border-color: $color-axa-blue;
+
+      &::after {
+        border-color: $color-axa-blue;
+      }
+    }
+  }
+
+  .a-radio__wrapper {
+    position: relative;
+    display: -webkit-inline-box;
+    display: -ms-inline-flexbox;
+    display: inline-flex;
+    min-height: 20px;
+    -webkit-box-align: center;
```

Comment:
should come automatically from our autoprefixer. Please remove it and test it if it works

Reply:
Yeah, they are all the result of sloppy copy-paste of v1 CSS. Will clean up...

Reply:
done